datumaro/plugins/voc_format/extractor.py

```diff
@@ -83,8 +96,9 @@ def _load_subset_list(self, subset_path):
                         if len(objects) == 3:
                             line = objects[1]
                         else:
-                            raise ValueError(
-                                "Line %s: unexpected number " "of quotes in filename" % line
+                            raise InvalidAnnotationError(
+                                f"{osp.basename(subset_path)}:{i}: "
```

Comment:
```suggestion
                                f"{osp.basename(subset_path)}:{i + 1}: "
```

Lines are usually numbered starting with 1.

Reply:
Applied here and in other cases